Explorer/Assets/DCL/Character/CharacterCamera/Systems/ControlCinemachineVirtualCameraSystem.cs

```diff
@@ -95,70 +95,73 @@ private void SetActiveCamera(ICinemachinePreset cinemachinePreset, in CameraComp
                     // Disable Player input
                     inputMapComponent.Active &= ~InputMapComponent.Kind.Player;
 
+                    SetDefaultFreeCameraPosition(in cinemachinePreset);
+
                     break;
             }
         }
 
         [Query]
         private void HandleZooming(ref CameraComponent cameraComponent, ref CameraInput input, ref ICinemachinePreset cinemachinePreset, ref CinemachineCameraState state)
         {
```

Comment:
The only if I added is the [if (cameraComponent.CameraInputChangeEnabled)](https://github.com/decentraland/unity-explorer/blob/b02c5e11915d4d835f94afcafea0b3e62534ca34/Explorer/Assets/DCL/Character/CharacterCamera/Systems/ControlCinemachineVirtualCameraSystem.cs#L107); All the code inside that conditional is almost exactly as it already was implemented originally by Misha to have the input scrolling working with the camera mode changes based on input.

Since this PR already contains a lot of changes I prefer leaving that original code as it is right now and in any case change it in a different independent PR please.